### PR TITLE
SLI-1232 Pass icon as optional in notification options

### PR DESCRIPTION
**Pull request by eray-felek-sonarsource**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/ui/vulnerabilities/TaintVulnerabilitiesPanel.kt [hunk 1]
```diff
@@ -148,7 +149,7 @@ class TaintVulnerabilitiesPanel(private val project: Project) : SimpleToolWindow
             RefreshTaintVulnerabilitiesAction(),
             sonarLintActions.includeResolvedTaintVulnerabilitiesAction(),
             sonarLintActions.configure(),
-            OpenInBrowserAction(LEARN_MORE, "Learn more about taint vulnerabilities in SonarLint", TAINT_VULNERABILITIES_LINK)
+            OpenInBrowserAction(LEARN_MORE, "Learn more about taint vulnerabilities in SonarLint", TAINT_VULNERABILITIES_LINK, AllIcons.Actions.Help)
```
Comment (nquinquenel) [inline src/main/java/org/sonarlint/intellij/ui/vulnerabilities/TaintVulnerabilitiesPanel.kt:152]: You can remove this icon as well, we don't want it anymore in any places

File: src/main/java/org/sonarlint/intellij/ui/SecurityHotspotsPanel.java [hunk 2]
```diff
@@ -139,7 +140,7 @@ private static DefaultActionGroup createActionGroup() {
     actionGroup.add(sonarLintActions.filterSecurityHotspots());
     actionGroup.add(sonarLintActions.includeResolvedHotspotAction());
     actionGroup.add(sonarLintActions.configure());
-    actionGroup.add(new OpenInBrowserAction("Learn More", "Learn more about Security Hotspots in SonarLint", SECURITY_HOTSPOTS_LINK));
+    actionGroup.add(new OpenInBrowserAction("Learn More", "Learn more about Security Hotspots in SonarLint", SECURITY_HOTSPOTS_LINK, AllIcons.Actions.Help));
```
Comment (nquinquenel) [inline src/main/java/org/sonarlint/intellij/ui/SecurityHotspotsPanel.java:143]: same

File: src/main/java/org/sonarlint/intellij/actions/OpenInBrowserAction.kt [hunk 3]
```diff
@@ -19,12 +19,12 @@
...
 package org.sonarlint.intellij.actions
 
-import com.intellij.icons.AllIcons
 import com.intellij.ide.BrowserUtil
 import com.intellij.openapi.actionSystem.AnActionEvent
+import javax.swing.Icon
 
-class OpenInBrowserAction(text: String, description: String?, private val link: String) : AbstractSonarAction(
-    text, description, AllIcons.Actions.Help
+class OpenInBrowserAction(text: String, description: String?, private val link: String, icon: Icon?) : AbstractSonarAction(
```
Comment (nquinquenel) [inline src/main/java/org/sonarlint/intellij/actions/OpenInBrowserAction.kt:26]: You could put a default value `, icon: Icon? = null` so you don't have to add `null` everywhere

File: src/main/java/org/sonarlint/intellij/ui/vulnerabilities/TaintVulnerabilitiesPanel.kt (hunk 1 above)
Comment (eray-felek-sonarsource) [reply src/main/java/org/sonarlint/intellij/ui/vulnerabilities/TaintVulnerabilitiesPanel.kt:152]: It is displayed in Taint Vulnerabilities tab not in notification. I think we should keep it

File: src/main/java/org/sonarlint/intellij/ui/SecurityHotspotsPanel.java (hunk 2 above)
Comment (eray-felek-sonarsource) [reply src/main/java/org/sonarlint/intellij/ui/SecurityHotspotsPanel.java:143]: same for Security Hotspot

File: src/main/java/org/sonarlint/intellij/ui/vulnerabilities/TaintVulnerabilitiesPanel.kt (hunk 1 above)
Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/ui/vulnerabilities/TaintVulnerabilitiesPanel.kt:152]: You're right, good point!

**Review by nquinquenel — APPROVED** [review]

**Pull request merged**